src/main/java/com/smartcar/sdk/AuthClient.java

```diff
@@ -472,13 +472,17 @@ public Auth exchangeRefreshToken(String refreshToken) throws SmartcarException {
   }
 
   /**
-   * Determine if a vehicle is compatible with Smartcar given scopes.
-   * Vin and Scopes are required parameters
+   * Determine if a vehicle is compatible with the Smartcar API and the provided permissions.
+   * A compatible vehicle is a vehicle that: <br>
```

Comment:
do we need <br> here. i think it's weird to place that comments. don't think it's necessary, should break like on its own

Reply:
I just tested and it won't line break without them. I think it's because the description is all placed in a paragraph that doesn't have any other formatting

Reply:
try adding an indent in the comments.

Reply:
no good :(

Reply:
damn, you actually do have add html in the docs. wow.

anyway, recommended style is to use <ol> and <li>.

```
Unordered and ordered lists in comments should be represented using <ul> or <ol> respectively. The <ul> and </ul> or <ol> and </ol> tags should each be on a line of their own. List items should each be placed on their own line, and the <li> and </li> tags should be on their own line immediately before and after the item text.
```

source - https://dev.liferay.com/participate/advanced-javadoc-guidelines

so ugly :(